src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const arr = sourceString.split(';');
+  const result = {};
+
+  arr.forEach((style) => {
+    if (style.trim()) {
+      const [key, value] = style.split(':');
```

Comment:
After splitting by ':', it's a good practice to limit the number of splits to 2, in case the value itself contains ':'. You can do this by passing a second argument to `split`, like `style.split(':', 2)`.